.github/workflows/api_tests.yml

```diff
@@ -16,6 +16,7 @@ on:
     branches: [main]
 
 env:
+  RUBYOPT: --disable=frozen_string_literal
```

Comment:
We run our specs with `-f d --warnings` which emits warnings. Ruby 3.4 now warns against modifying strings because apparently in Ruby 4 (next year?) they will enable frozen strings by default